src/Pages/SubscribeList/SubscribeDetail/Components/DetailOption/DetailOption.js

```diff
@@ -0,0 +1,277 @@
+              </thead>
+              <tbody>
+                <tr className="optionRow1">
+                  <th>
+                    <span className="contents">구독옵션</span>
+                  </th>
+                  <td>
+                    <select
+                      value={subscribeOption}
+                      onChange={this.changeSubOption}
+                    >
+                      <label htmlFor="option">구독 옵션</label>
+                      <option className="selected" value="">
+                        구독기간을 선택해주세요
+                      </option>
+                      <option className="option" value="정기구독">
+                        정기결제 (2주마다 자동결제)
+                      </option>
+                      <option className="option" value="1회 무료체험">
+                        1회 무료 체험(+ 배송비 3,000원)
+                      </option>
+                    </select>
+                  </td>
+                </tr>
+                <tr className="optionRow2">
+                  <th>
+                    <span className="contents">수량</span>
+                  </th>
+                  <td>
+                    <div className="detailAmount">
+                      <button
+                        type="button"
+                        className="btnAmount"
+                        onClick={this.minusCount}
+                      >
+                        -
+                      </button>
+                      <span>{orderCount}</span>
+                      <button
+                        type="button"
+                        className="btnAmount"
+                        onClick={this.addCount}
+                      >
+                        +
+                      </button>
+                    </div>
+                  </td>
+                </tr>
+                <tr className="optionRow3">
+                  <th>
+                    <span className="contents">편지 추가</span>
+                  </th>
+                  <td>
+                    <form className="letterCheckboxes">
+                      <div className="letterCheckbox">
+                        <input
+                          className="contents"
+                          id="letter"
+                          type="radio"
+                          name="letter"
+                          value="letter"
+                          checked={this.state.hasLetter ? true : false}
+                          onClick={this.chkHasLetter}
+                          readOnly
+                        />
+                        <label for="letter">추가할게요(+2,500)</label>
+                      </div>
+                      <div className="letterCheckbox">
+                        <input
+                          className="contents"
+                          id="letterNo"
+                          type="radio"
+                          name="letter"
+                          value="noLetter"
+                          checked={this.state.hasLetter ? false : true}
```

Comment:
삼항연산자를 사용할 필요없이 불리안 값을 이용해 작성해주셔도 되겠네요!
```suggestion
                          checked={!this.state.hasLetter}
```